go/pkg/rexec/rexec.go

```diff
@@ -419,27 +419,21 @@ func (ec *Context) GetOutputFileDigests(useAbsPath bool) (map[string]digest.Dige
 // This function only returns the digests of output files and output directories.
 // GetOutputFileDigests will return the digest of output files as well as the digest of all the files
 // that exist under the output directories.
+// This function is supposed to be run after a successful cache-hit / remote-execution
+// has been run with the given execution context. If called before the completion of
+// remote-execution, the function returns an error.
 func (ec *Context) GetOutputDigests() (outputFileDigests map[string]digest.Digest, outputDirectoryDigests map[string]digest.Digest, err error) {
```

Comment:
Looking at it now, `GetOutputDigests` just passes forward `ec.respPb.OutputFiles/OutputDirectories`.

It may make more sense to just remove this and instead add `OutputDirectoriesDigests` to the `ec.Metadata`? (since it already has `OutputDigests`.

Reply:
So we discussed this. We do want to separate out OutputFile digests and OutputDir digests and that means ec.Metadata now will have three things - OutputDigests, OutputFileDigests and OutputDirDigests. Given that, it seemed easier to add a function to return this info instead of sideloading it through the Metadata.

Reply:
As discussed in the meeting, I changed the OutputDigests field to OutputFileDigests and added a new OutputDirectoryDigests field.